src/components/Popover/Popover.jsx

```diff
@@ -1,20 +1,23 @@
 import Tippy from "@tippyjs/react";
 import clsx from "clsx";
 import PropTypes from "prop-types";
-import React, { Children } from "react";
+import React, { Children, useState } from "react";
 import { followCursor } from "tippy.js";
 import "tippy.js/dist/border.css";
 import "tippy.js/dist/tippy.css";
 import { getChildrenByType } from "../../helpers/children";
 import styles from "./Popover.module.css";
 import scrollFix from "./PopoverScroll.module.css";
 
-export const Popover = ({ className, children, skidding = 0, distance = 10, ...rest }) => {
+export const Popover = ({ isLazy = true, skidding = 0, distance = 10, className, children, ...rest }) => {
```

Comment:
By default this is true, so all new popovers are lazy. We should have a companion PR to pass this flag as false wherever we don't have appropriate automation coverage or wherever manual testing effort is too much.